Build failure from TCT

**Describe the bug**

```
~/c/p/penumbra (custody-protocol|✚1…) $ cargo run --release --bin pd -- generate-testnet
   Compiling pd v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/pd)
   Compiling penumbra-proto v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/proto)
   Compiling penumbra-crypto v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/crypto)
   Compiling penumbra-tct v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/tct)
   Compiling penumbra-transaction v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/transaction)
   Compiling penumbra-storage v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/storage)
   Compiling penumbra-chain v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/chain)
   Compiling penumbra-component v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/component)
   Compiling penumbra-shielded-pool v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/shielded-pool)
   Compiling penumbra-ibc v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/ibc)
   Compiling penumbra-stake v0.1.0 (/Users/hdevalence/code/penumbra/penumbra/stake)
error[E0275]: overflow evaluating the requirement `penumbra_tct::internal::complete::node::children::Children<penumbra_tct::internal::complete::leaf::Leaf<penumbra_tct::internal::complete::item::Item>>: Send`
  |
  = help: consider increasing the recursion limit by adding a `#![recursion_limit = "256"]` attribute to your crate (`pd`)
  = note: required because it appears within the type `penumbra_tct::internal::complete::node::Node<penumbra_tct::internal::complete::leaf::Leaf<penumbra_tct::internal::complete::item::Item>>`
  = note: required because it appears within the type `penumbra_tct::internal::complete::node::children::shape::___C<penumbra_tct::internal::complete::node::Node<penumbra_tct::internal::complete::leaf::Leaf<penumbra_tct::internal::complete::item::Item>>>`
  = note: required because of the requirements on the impl of `Send` for `Unique<penumbra_tct::internal::complete::node::children::shape::___C<penumbra_tct::internal::complete::node::Node<penumbra_tct::internal::complete::leaf::Leaf<penumbra_tct::internal::complete::item::Item>>>>`
  = note: required because it appears within the type `Box<penumbra_tct::internal::complete::node::children::shape::___C<penumbra_tct::internal::complete::node::Node<penumbra_tct::internal::complete::leaf::Leaf<penumbra_tct::internal::complete::item::Item>>>>`
  = note: required because it appears within the type `penumbra_tct::internal::complete::node::children::Children<penumbra_tct::internal::complete::node::Node<penumbra_tct::internal::complete::leaf::Leaf<penumbra_tct::internal::complete::item::Item>>>`

For more information about this error, try `rustc --explain E0275`.
error: could not compile `pd` due to previous error
```

--- a/pd/src/main.rs
+++ b/pd/src/main.rs
@@ -1,4 +1,5 @@
 #![allow(clippy::clone_on_copy)]
+#![recursion_limit = "256"]
 use std::{
     net::{Ipv4Addr, SocketAddr},
     path::PathBuf,

